### PR TITLE
docs: add license and readme

### LINKED ISSUE #12: Before Moving to Express org
Hey @bjohansebas and @kjugi this came up in the recent TC meeting, @kjugi was there before we kicked him out to talk security

We can move this into express org if yall are open to it and when you're ready.

At a minimum the project should have:

- [ ] Readme file, explaining how to use the codemods
- [ ] License file

### PR DESCRIPTION
**Pull request by bjohansebas**
The necessary documentation is added to be moved to the Express organization

closes #12

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,23 @@
+(The MIT License)
+
+Copyright (c) 2024 Sebastian Beltran <bjohansebas@gmail.com>
+Copyright (c) 2024 Filip Kudla <kjugi@users.noreply.github.com>
```
Comment (bjohansebas) [inline LICENSE:4]: @kjugi change the email if you want

Comment (kjugi) [reply LICENSE]: done, thanks for ping 👍

**Comment by bjohansebas** [comment]
I added a workflow to automate the use of the package, so it stays updated after each PR. It will be useful when others want to add more options but don't update the README

File: README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# @expressjs/codemod
+
+[![NPM Version][npm-version-image]][npm-url]
+[![NPM Install Size][npm-install-size-image]][npm-install-size-url]
+[![NPM Downloads][npm-downloads-image]][npm-downloads-url]
+[![OpenSSF Scorecard Badge][ossf-scorecard-badge]][ossf-scorecard-visualizer]
```
Comment (bjohansebas) [inline README.md:6]: This will not display correctly yet, but once it's published and moved to Express, it will be displayed properly.

**Comment by bjohansebas** [comment]
@kjugi If you think the texts can be improved, feel free to change them

**Review by kjugi — CHANGES_REQUESTED** [review]
small comments

File: .github/workflows/generate-readme.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build package
+        run: npm run build
+
+      - name: Run script
+        working-directory: .github/actions
+        run: node ./generate-help-docs.mjs
```
[the remark this answers is not in the document]
Comment (bjohansebas) [reply .github/workflows/generate-readme.yml:31]: good idea

File: .github/workflows/generate-readme.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Create Pull Request
+        uses: gr2m/create-or-update-pull-request-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+            title: 'Update options in README'
```
[the remark this answers is not in the document]
Comment (bjohansebas) [reply .github/workflows/generate-readme.yml:40]: I suppose the change I made is enough, right?

File: .github/actions/generate-help-docs.mjs [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+})
+
+output = output.replace(
+  /(?<=Usage: @expressjs\/codemod \[codemod\] \[source\] \[options\]\n\n)[\s\S]*?(?=Options:)/,
+  '',
+)
+
+const readmePath = join('..', '..', 'README.md')
+const readme = await readFile(readmePath, 'utf-8')
+
```
[the remark this answers is not in the document]
Comment (bjohansebas) [reply .github/actions/generate-help-docs.mjs:16]: It's a small script, I don't see the need to do this check

File: .github/workflows/generate-readme.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (kjugi) [reply .github/workflows/generate-readme.yml:40]: I'm fine when PR is created only when we change file from list 👌🏻

[the remark this answers is not in the document]
Comment (bjohansebas) [reply .github/workflows/generate-readme.yml:40]: In fact, it works that way now, when something in `index.ts` is changed, the workflow will run, and if that change affects the output, a PR will be opened.

**Review by kjugi — APPROVED** [review]

**Comment by bjohansebas** [comment]
@kjugi I'm going to send the request to move this repository to Express.

**Pull request merged**